source/index.txt

```diff
@@ -1 +1,55 @@
-Test Build
+.. _visual-studio-extension:
+
+.. _visual-studio-extension-landing:
+
+===============================
+{+product+}
+===============================
+
+.. default-domain:: mongodb
+
+.. toctree::
+   :titlesonly:
+   :maxdepth: 1
+
+   Installation </install>
+   Builders </builders>
+   LINQ </linq>
+   Configuration </configuration>
+   FAQ </faq>
+   What's New </whats-new>
+
+Overview
+--------
+
+The `{+product+} <{+product-home-page+}>`__ is a NuGet package
```

Comment:
Maybe something like: "is a Roslyn analyzer that..." or "is a Roslyn analyzer (packaged as a NuGet package) that..."

Reply:
I agree "NuGet package" should be swapped. I'm nervous mentioning "Roslyn Analyzer" in the introductory sentence of the guide as I think it may make readers think they need to understand "Roslyn" to use the MongoDB Analyzer (I'm not sure they need to understand the Roslyn compiler to use the MongoDB Analyzer but I could be mistaken).

Would you be comfortable replacing "NuGet package" with the word "tool"? Based off the [Sarif specification](https://github.com/microsoft/sarif-tutorials/blob/main/docs/1-Introduction.md#about-static-analysis-tools), it seems like it is appropriate to refer to a static analysis tool as a "tool":

> We often refer to a static analysis tool simply as a "tool."

If you think "Roslyn Analyzer" is a better term than "tool" however I'll use "Roslyn Analyzer". Just wanted to raise my concern about introducing "Roslyn" too soon.

Reply:
I'll use "Roslyn Analyzer" for the time being in order to merge this PR so that I can get the next page into review. When you have a free moment, let me know your thoughts on the term "tool" and if you like the switch I'll add make the change as part of a different PR.

Reply:
Discussed on slack, and we agreed "tool" is acceptable here.